frontend/express/public/javascripts/countly/countly.views.js

```diff
@@ -1482,19 +1482,6 @@ window.ManageAppsView = countlyView.extend({
             }
         }
 
-        /*
-        * Prevent text highlights
-        */
-        $("#clear-data > li").on("selectstart", function(e) {
-            e.preventDefault();
-        });
-        $("#clear-app-data").on("selectstart", function(e) {
-            e.preventDefault();
-        });
-        $("#delete-app").on("selectstart", function(e) {
-            e.preventDefault();
-        });
-
```

Comment:
Any reason why these were removed?
I understand they were replaces with other button ids?
But maybe the new buttons need the same?
Just rechecking

Reply:
You're right, these should be adapted to the new menu buttons, on it 👍